src/Microsoft.DotNet.Interactive.PowerShell.Tests/PowerShellKernelTests.cs

```diff
@@ -216,5 +217,27 @@ public async Task VerifyAllUsersProfileRuns()
                 File.Delete(_allUsersCurrentHostProfilePath);
             }
         }
+
+        [Fact]
+        public async Task PowerShell_Custom_Object_OutDisplay_Passed()
```

Comment:
I would rename this. All tests "should pass" so "passed" is not adding information.

What specifically can we expect about the output?